db/migrations.go

```diff
@@ -43,13 +43,14 @@ func baselineSchemaManualMigrations(tx *gorm.DB) error {
 }
 
 var Migrations []*gormigrate.Migration = []*gormigrate.Migration{
-	// {
-	// 	ID: "2023060800", // Set to todays date, starting with 00 for first migration
-	// 	Migrate: func(tx *gorm.DB) error {
-	// 		guardMigration("2023060800")
-	// 		return tx.Migrator().AddColumn(&ReplicationConstraint{}, "RC")
-	// 	},
-	// },
+	{
+		ID: "2024020900", // Set to todays date, starting with 00 for first migration
+		Migrate: func(tx *gorm.DB) error {
+			guardMigration("2024020900")
+
+			return tx.AutoMigrate(&SPAttribute{}, &TenantsSPs{})
+		},
+	},
```

Comment:
@stephen-pl  since we don't actually have a prod db deployed anywhere, this migration is not strictly necessary as we don't have anything to migrate (other than our local dev environments). I may end up removing it before merge, but it means we'll need to delete & re-create your local dev db to re-apply the newly updated baseline migration.

Also, we should store an up-to-date dump of some dev db data so we can easily import it for testing purposes

Reply:
Sounds good. Do you want to set up something like [ElephantSQL](https://www.elephantsql.com/plans.html) for a cloud db in the short term?

Reply:
should be ok we'll deploy soon enough https://github.com/data-preservation-programs/spade/issues/56